### DISCUSSION TITLE
Selecting from Measure groups

### DISCUSSION THREAD

**Discussion by danbuntu** [DaxStudio/DaxStudio#676] [Q&A]
Hello

I've got a power bi dashbaord that uses a measures group. ie. I've gone to Modelling > new table anf given the new table a name. I've then created measures in that table and removed the default blank row. I do this as a way to organise my measures in one place. 

When I connect to the dahsboard with Dax studio I'm unable to select any of the measures from this measrues group table in the editor. All the other tables that have been created from sql quries are working fine autocomplete etc is working fine.

I feel like I must be missing something here.

**Comment by dgosbell** [comment]
Can you upload an example .pbix file with an example of this structure to this discussion on github. I've tested with a model which I think is similar to what you have described and it works fine.

And can you expand on what you mean by "I'm unable to select any of the measures from this measrues group table in the editor"? Are you trying to drag and drop from the metadata pane onto the editor. Or are you trying to drag into the Query Builder. Or are you saying that when you type a `[` character that the measures do not appear in the intellisense list?

**Reply by danbuntu** [reply]
PBIX attached

There are two sets of data in it.

AgeBands - A 'normal' dataset that been created from a sql query and has the standard 'table' icon. This works as epexcted in Dax Studio. I can drag and drop columns and if I can use autocomplete with intellisense when typing to select columns.

The second set 'Static numbers' has the icon for 'measures group'. I created it my going into power query editor and clicking 'new data' and giveing it a number. I then create a measure in it and deleted then deleted the original blank row. In Dax Studio typing 'Static' brings upthe table name in intellisense. Adding a [ though then doen't bring up the list of columns it to sleect from. If I manually type sum(staticnumbers[6000]) and run it I get the error

'Column '6000' in table staticnumbers cannot be found or may not be used in this expression'
[example_dashboard.zip](https://github.com/DaxStudio/DaxStudio/files/6816213/example_dashboard.zip)

**Reply by danbuntu** [reply]
And has expected - I'm a numpty and I was missing something.

I'm tring to reference the measure by table[measurename]
and this is wrong I just need to reference is by [measurename] and job done - bacuse a masurename can only exist once in the whole model.


Thank you for time @dgosbell as acutally working thorugh this has helped me understand it better.
